### ISSUE TITLE
Code generation fails w/ custom output path

### ISSUE THREAD

**Issue by vogelb** [os72/protoc-jar-maven-plugin#71]
Hi there,

I used the multiple output targets gRPC example and set an outputDirectory for each target.

Compilation with the following exception (writing success file fails).
After looking at the code, I found a workaround setting optimizeCodegen to false.

```
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.github.os72:protoc-jar-maven-plugin:3.7.0.1:run (default) on project rp-template1-grpc: File operation failed

Caused by: java.io.IOException: Das System kann den angegebenen Pfad nicht finden (File not found)
    at java.io.WinNTFileSystem.createFileExclusively (Native Method)
    at java.io.File.createNewFile (File.java:1012)
    at com.github.os72.protocjar.maven.ProtocJarMojo.execute (ProtocJarMojo.java:353)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
```

```
<plugin>
        <groupId>com.github.os72</groupId>
        <artifactId>protoc-jar-maven-plugin</artifactId>
        <version>3.7.0.1</version>
        <executions>
                <execution>
                        <phase>generate-sources</phase>
                        <goals>
                                <goal>run</goal>
                        </goals>
                        <configuration>
                                <protocArtifact>com.google.protobuf:protoc:3.0.0</protocArtifact>
                                <!-- <optimizeCodegen>false</optimizeCodegen>-->
                                <inputDirectories>
                                        <include>src/main/proto</include>
                                </inputDirectories>
                                <outputTargets>
                                        <outputTarget>
                                                <type>java</type>
                                                <outputDirectory>generated-sources/model</outputDirectory>
                                        </outputTarget>
                                        <outputTarget>
                                                <type>grpc-java</type>
                                                <pluginArtifact>io.grpc:protoc-gen-grpc-java:1.19.0</pluginArtifact>
                                                <outputDirectory>generated-sources/service</outputDirectory>
                                        </outputTarget>
                                </outputTargets>
                        </configuration>
                </execution>
        </executions>
</plugin>
```

**Comment by os72** [comment]
I assume this is not a 'clean' run (`mvn install` or so)?
`mvn clean install` should work though; does it?

**Comment by os72** [comment]
Could you try 3.7.0.2-SNAPSHOT?

**Comment by os72** [comment]
Fixed in version 3.7.0.2
